src/extension/intellisenseHelper.ts

```diff
@@ -7,12 +7,27 @@ import * as os from "os";
 import * as path from "path";
 import * as Q from "q";
 import * as vscode from "vscode";
+import * as semver from "semver";
 import {Telemetry} from "../common/telemetry";
 import {TelemetryHelper} from "../common/telemetryHelper";
+import {CommandExecutor} from "../common/commandExecutor";
 import {TsConfigHelper} from "./tsconfigHelper";
 import {SettingsHelper} from "./settingsHelper";
+import {Log, LogLevel} from "../common/log";
+
+
+interface InstallProps {
```

Comment:
For some reason Github marked my previous comment as outdated... Here it is again:

According to our convention, I think this should be **I**InstallProps (missing an `I`)